kytos/utils/config.py

```diff
@@ -58,46 +59,33 @@ def set_env_or_defaults(self):
         If no environment variable is found and the config section/key is
         empty, then set some default values.
         """
-        napps_api = os.environ.get('NAPPS_API_URI')
-        napps_repo = os.environ.get('NAPPS_REPO_URI')
-        user = os.environ.get('NAPPS_USER')
-        token = os.environ.get('NAPPS_TOKEN')
-        napps_path = os.environ.get('NAPPS_PATH')
+        Option = namedtuple('Option', ['section', 'name', 'env_var',
+                                       'default_value'])
+
+        options = [Option('auth', 'user', 'NAPPS_USER', None),
+                   Option('auth', 'token', 'NAPPS_TOKEN', None),
+                   Option('napps', 'api', 'NAPPS_API_URI',
+                          'https://napps.kytos.io/api/'),
+                   Option('napps', 'repo', 'NAPPS_REPO_URI',
+                          'https://napps.kytos.io/repo'),
+                   Option('kytos', 'api', 'KYTOS_API',
+                          'http://localhost:8181/')]
+
+        for option in options:
+            if not self.config.has_option(option.section, option.name):
+                env_value = os.environ.get(option.env_var)
+                if env_value and not hasattr(self.config, options):
+                    self.config.set(option.section, option.name, env_value)
+                elif option.default_value:
+                    self.config.set(option.section, option.name,
+                                    option.default_value)
```

Comment:
Also, I have just thought that we can reduce the code here with:
```python
for option in options:
    if not self.config.has_option(option.section, option.name):
        env_value = os.environ.get(option.env_var, option.default_value)
        if env_value and not hasattr(self.config, options):
            self.config.set(option.section, option.name, env_value)
```
(Consider that I haven't understood the `hasattr(self.config, options)` condition....)